docs/user-guide/src/SUMMARY.md

```diff
@@ -20,7 +20,7 @@
     - [Upgrade]()
     - [Features]()
         - [Automated cleaning API]()
-        - [External Remediation]()
+        - [Remediation Controller](capm3/remediaton.md)
```

Comment:
Since it is a features list, maybe no need to specify the controller name here?

Reply:
Thanks for review. Done